manifests/chloride.pp

```diff
@@ -0,0 +1,12 @@
+# Manage the chloride gem requirement for the bulk agent installer
+class pe_bulk_agent_install::chloride (
+  Optional['String'] $gem_source = undef,
+) {
+
+  package {'chloride':
+    ensure   => 'present',
+    provider => 'puppet_gem',
```

Comment:
This will lock this class into Puppet 4 ( aside for the type syntax already present, however this is mostly to say maybe it should be a param, just in case )